src/main/java/bookstore/model/Book.java

```diff
@@ -0,0 +1,22 @@
+package bookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
```

Comment:
add required constraints according to the task description 

Book entity with fields:

id (Long, PK)
title (String, not null)
author (String, not null)
isbn (String, not null, unique)
price (BigDecimal, not null)
description (String)
coverImage (String)